### PR TITLE
add margin in tags of event cards

### LINKED ISSUE #2346: Add bottom margin to event tags in event card
**Describe the bug**

The tags in event cards shown on homepage are quite close to the bottom boundary of the card. It will be better to add a bottom margin.
Also in the responsive mode, the tags go out of cards.
**To Reproduce**
1. Go to '/'
2. See error

**Expected behavior**

There should be some gap between tags and bottom boundary of card
**Screenshots**

Screenshot from 2019-03-20 15-47-08

Screenshot from 2019-03-20 16-12-58

**Additional context**

I am working on it.

### PR DESCRIPTION
**Pull request by khushishikhu**
Fixes #2346 
Capture1

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by khushishikhu** [comment]
Capture1
Capture2

**Comment by khushishikhu** [comment]
sir i have tried to fix this issue.Please review the changes.

File: app/templates/index.hbs [hunk 1]
```diff
@@ -11,7 +11,7 @@
   {{/if}}
   {{#if this.filteredEvents}}
     <h2 class="main-heading">{{t 'Upcoming Events'}}</h2>
-    <div class="ui stackable three column grid">
+    <div class="ui stackable three column grid filtered">
```
Comment (iamareebjamal) [inline app/templates/index.hbs:14]: The bug is in event card, not in upcoming section. This is the 5th time I'm repeating this

Comment (khushishikhu) [reply app/templates/index.hbs:14]: i apologize sir. i thought that the event card is under upcoming section so i was trying to make it in those cards only.

**Comment by mariobehling** [comment]
This PR has been inactive for over a month. Therefore i am closing it.

**Pull request closed without merging**